### DISCUSSION TITLE
Officially host SearX.

### DISCUSSION THREAD

**Discussion by beedell-roke** [searx/searx#3179] [Ideas]
As is demonstrated by 'http://github.com/searx/searx/discussions/3178', I want to utilise SearX because SearX appears to be significantly superior to any search-engine that I have previously utilised, but dissimilarly to most search engines, when I attempted to choose utilise it, too many possible options were presented to me. Consequently, I was not easily able to chooce, and not least additionally because the metadata was not assistive during choice of which to utilise because the presentation of the versions of the instances is not consistent.

However, despite this obvious difficulty, the current lack of official hostage was confirmed by 'http://github.com/searx/searx/discussions/3178#discussioncomment-2300646'. I believe that official hostage is able to not solely improve the experience for new utilisers of SearX, but additionally allow the developers of SearX to understand what is useful for all that are hosting SearX more well than the developers are currently able to.

Consequently, please consider official hostage of SearX so that the community is able to easily recommend and collaborate to improve 1 version whilst simultaneously allow and innately support divergence/difference. I am thankful for any consideration.

**Comment by unixfox** [comment]
There is no plan to do that.

There used to be one official searx instance but asciimoo closed it due to the lack of free time.

**Reply by beedell-roke** [reply]
That is reasonable, but I am confident that what I have stated would be incredibly beneficial. Which instance is your preference?

**Reply by unixfox** [reply]
I'm sorry I can't really recommend you an instance. You should choose the one that is working great for you on https://searx.space
